src/renderer/webtorrent.js

```diff
@@ -54,11 +54,16 @@ const VERSION_STR = VERSION.match(/([0-9]+)/g)
  */
 const VERSION_PREFIX = '-WD' + VERSION_STR + '-'
 
+/**
+ * Generate an ephemeral peer ID each time.
+ * TODO: once there are around 2^24 = ~8 million WebTorrent Desktops online at the same time,
+ *       ID collisions will start happening. Birthday paradox. Can we use more than six bytes?
```

Comment:
Yeah this is fine. I'll just change the comment to say that peer ID collisions aren't that bad.